lib/libzfs/libzfs_dataset.c

```diff
@@ -2923,6 +2923,18 @@ zfs_prop_get(zfs_handle_t *zhp, zfs_prop_t prop, char *propbuf, size_t proplen,
 		zcp_check(zhp, prop, val, NULL);
 		break;
 
+	case ZFS_PROP_PBKDF2_ITERS:
```

Comment:
It looks like we could add `ZFS_PROP_PBKDF2_ITERS` after the `ZFS_PROP_OBJSETID` above so it's handled the same way as those properties and the full value is always printed.